ui/src/main/java/org/pentaho/di/ui/spoon/trans/TransGraph.java

```diff
@@ -4277,6 +4277,8 @@ private synchronized void prepareTrans( final Thread parentThread, final String[
       @Override
       public void run() {
         try {
+
+          trans.setInitialLogBufferStartLine();
```

Comment:
Why you need this here? The method execute on trans is not call?

Reply:
Some transformations aren't started from the execute method